### PR TITLE
persian translation files 

### PR DESCRIPTION
**Pull request by wikiccu**
about issue :  #595 
## Description
I just translated locale files for the Persian language

special thanks to all members for building this great project <3

### CODE REVIEW

File: resources/lang/fa/pagination.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    | Pagination Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the paginator library to build
+    | the simple pagination links. You are free to change them to anything
+    | you want to customize your views to better match your application.
+    |
+    */
+
+    'previous' => '&laquo; قبلی',
```
Comment (moslem-deris) [inline resources/lang/fa/pagination.php:15]: This is a translation file, please only enter translated phrases, words and sentences and avoid using HTML character codes

File: resources/lang/fa/validation.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => ':attribute باید پذیرفته شده باشد.',
+    'active_url'           => 'آدرس :attribute معتبر نیست.',
+    'after'                => ':attribute باید تاریخی بعد از :date باشد.',
+    'after_or_equal'       => ':attribute باید تاریخی بعد از :date، یا مطابق با آن باشد.',
+    'alpha'                => ':attribute باید فقط حروف الفبا باشد.',
+    'alpha_dash'           => ':attribute باید فقط حروف الفبا، اعداد، خط تیره و زیرخط باشد.',
```
Comment (moslem-deris) [inline resources/lang/fa/validation.php:20]: In my opinion instead of `و زیرخط` its better to say `یا زیرخط`

**Review by moslem-deris — CHANGES_REQUESTED** [review]

File: resources/lang/fa/validation.php (hunk 2 above)
Comment (Mohammad-Alavi) [reply resources/lang/fa/validation.php:20]: I agree with this change. Please implement it so we can merge this PR.

**Comment by moslem-deris** [comment]
@wikiccu Check review comments and apply needed changes so we can merge your PR, thank you.

File: resources/lang/fa/validation.php (hunk 2 above)
Comment (Mohammad-Alavi) [reply resources/lang/fa/validation.php:20]: Ill fix this small issue after merge.

**Pull request merged**